run_tribler.py

```diff
@@ -1,6 +1,7 @@
 import os
 import sys
 import multiprocessing
+import logging  # TODO: Remove this import
```

Comment:
If you don't like all this logging, maybe keep these edits and add it to your gitignore?